### PR TITLE
The @hasrole directive works

### PR DESCRIPTION
**Pull request by drbyte**
Screen Shot 2019-05-13 at 10 41 13 AM

Screen Recording 2019-05-13 at 10 42 AM

### CODE REVIEW

**Comment by elodiecmpn16** [comment]
Thanks a lot!! i'll follow your advices!

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -12,14 +12,14 @@
                         @csrf
 
                         <div class="form-group row">
-                            <label for="username" class="col-md-4 col-form-label text-md-right">{{ __('E-Mail Address ou nom d utilisateur') }}</label>
+                            <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('E-Mail Address ou nom d utilisateur') }}</label>
```
Comment (elodiecmpn16) [inline resources/views/auth/login.blade.php]: i change that cause i'm using the username and not the email

File: resources/views/accueil.blade.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
         <div class="container">
             <div class="row accueilMetiers">
-                @if(auth()->user()->hasRole('admin'))
+                @role('admin')
```
Comment (elodiecmpn16) [inline resources/views/accueil.blade.php]: I wish i could do that but this is the point of my problem, i can't use those blade role directive

**Comment by drbyte** [comment]
My point of this pull request is twofold:
1. To demonstrate that the `@role` directive does work, in a new environment. Hence the gif showing the login and that the role-specific content only shows, based on the `@role` directive.

2. To show a few things in your demo app that I had to change to make your app work at all. Most of the changes, besides using the `@role` directives in the view, are unrelated to the `@role` matter. 
(It feels like this is a demo-copy, and not the real app code. That's fine.)

I suspect something "else" in your real app is causing your `@role` issue.

Perhaps rebuilding your `vendor` directory might be helpful? Just guessing, in case something old in there is conflicting somehow?
`rm -rf vendor && composer install`

**Comment by elodiecmpn16** [comment]
Thanks a lot!  Unfortunately my french prevents me to understand your first point about the gif...? actually this is a real app code i'm a newbie since 2 years and i developped this app alone for about almost 1 year! hope i will find the solution!  it seems crazy your worked on your time for me so thanks again for your time!! have a good one :)

**Comment by drbyte** [comment]
Pour plus de clarté, je vais essayer de ré-expliquer mon point sur le gif.

Le gif est simplement la preuve que le code fonctionne correctement. Dans mon environnement

Pour tenter de trouver la cause de votre problème, je devais être capable de le recréer. Par conséquent, j'ai essayé d'exécuter votre code et je l'ai modifié pour utiliser la syntaxe @role. Et, à ma grande surprise, cela a fonctionné!

Il n'y avait plus rien pour moi de déboguer.

Mais je voulais vous montrer que cela fonctionne, en utilisant votre code, afin que vous puissiez examiner d'autres causes telles que votre propre environnement.

**Comment by elodiecmpn16** [comment]
merci pour ces précisions! Je vais suivre vos conseils et chercher le problème coté environnement et non code! merci encore!

**Pull request merged**